pkgs/development/libraries/duckdb/default.nix

```diff
@@ -108,6 +108,7 @@ stdenv.mkDerivation (finalAttrs: {
     in
     ''
       runHook preInstallCheck
+      (($(ulimit -n) < 1024)) && ulimit -n 1024
```

Comment:
If `$(ulimit -n) < 1024` returns 1, won't the build fail directly?

Reply:
no.

https://www.gnu.org/software/bash/manual/bash.html#The-Set-Builtin
> -e
> ...
> The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, part of any command executed in a && or || list except the command following the final && or ||, any command in a pipeline but the last, or if the command’s return status is being inverted with !.
> ...

also note that the linux builds passed (where ulimit -n is 1024 i believe)